docs/user/aws/limits.md

```diff
@@ -0,0 +1,74 @@
+# AWS Limits
+
+You can find a comprehensive list of the default AWS service limits published here:
+
+[AWS Service Limits][service-limits]
+
+Below, we’ll identify OpenShift cluster needs and how those impact some of those limits.
+
+## S3
+
+There is a limit of 100 S3 buckets per account. The installation creates a bucket temporarily. Also, the registry 
+component creates a permanent bucket. This will limit the number of clusters per account to 99.
+
+## VPC
+
+Each cluster creates its own VPC. The default limit of VPCs per region is 5 and will allow 5 clusters. To have more 
+than 5 clusters, you will need to increase this limit.
+
+## Elastic Network Interfaces (ENI)
+
+The default installation creates 21 + the number of availability zones of ENIs (e.g. us-east-1 = 21 + 6 = 27 ENIs). 
+The default limit per region is 350. Additional ENIs are created for additional machines and elastic load balancers 
+created by cluster usage and deployed workloads. A service limit increase here may be required to satisfy the needs of 
+additional clusters and deployed workloads.
+
+## Elastic IP (EIP)
+
+For a single, default cluster, your account will have the needed capacity limits required. There is one exception, 
+“EC2-VPC Elastic IPs”. The installer creates a public and private subnet for each availability zone within a region to 
+provision the cluster in a highly available configuration. In each private subnet, a separate NAT Gateway is created and 
+requires a separate elastic IP. The default limit of 5 is sufficient for most regions and a single cluster. For the 
+us-east-1 region, a higher limit is required. For multiple clusters, a higher limit is required. Please see this map for 
+a current region map with availability zone count. We recommend selecting regions with 3 or more availability zones.
```

Comment:
Did you mean "this map" to be a link or something?

Reply:
Fixed. Thanks!